### DISCUSSION TITLE
chakra-ui/layout vs chakra-ui/react

### DISCUSSION THREAD

**Discussion by spidertwin2** [chakra-ui/chakra-ui#3631] [Q&A] [answered]
What is the difference between-

```
import { Box } from "@chakra-ui/layout"
import { Box } from "@chakra-ui/react"
```

thanks :bird:

**Comment by ViaxCo** [comment, marked as answer]
They are functionally the same. `@chakra-ui/react` exports everything, so importing `Box` from it is really importing it from `@chakra-ui/layout`
[Reference](https://github.com/chakra-ui/chakra-ui/blob/main/packages/react/src/index.tsx)
